Add-ons/UmbracoCourier/Architecture/DataResolvers.md

```diff
@@ -1,159 +1,157 @@
-	        <add key="Content All">contentAll</add>
-	    </contentPickers>
-	</macroPropertyTypeResolvers>
-
-### MediaPicker
-* **Full name:** `Umbraco.Courier.DataResolvers.MediaPicker`
-* **Triggers on:** Propertydata, which have a mediapicker as data type (configured in courier.config)
-* If value is set, and is an int, courier will convert the value to the media GUID, add the node as a dependency. On extraction the GUID will be converted back to the right ID.
-
-##### Configuration:
-
-	<mediaPickers>
-	    <!-- add new datatype elements for data types that stores media ids (ex: "1242" or "1726,2362,2323") -->
-	    <add key="mediaPicker">EAD69342-F06D-4253-83AC-28000225583B</add>
-	    <add key="damp2">ef94c406-9e83-4058-a780-0375624ba7ca</add>
-	</mediaPickers>
-
-
-### RelatedLinks
-* **Full name:** `Umbraco.Courier.DataResolvers.RelatedLinks`
-* **Triggers on:** Propertydata, which have a RelatedLinks type as data type
-* If values are set, courier will convert the values to the corresponding GUIDs, add the nodes as dependencies. On extraction the GUIDs will be converted back to the right IDs.
-
-
-### RTEstylesheets
-* **Full name:** `Umbraco.Courier.DataResolvers.RTEstylesheets`
-* **Triggers on:** The Rich text editor DataType
-* If the RTE have any stylesheets associated, these will be added as dependencies to the data type
-
-### Tags
-* **Full name:** `Umbraco.Courier.DataResolvers.Tags`
-* **Triggers on:**  Propertydata, which have a Tags type as data type
-* Selected tags are included as separate dependencies and extracted along with the document.
-
-### TemplateResources
-* **Full name:** `Umbraco.Courier.DataResolvers.TemplateResources`
-* **Triggers on:**  Templates
-* Detects linked images, JavaScript files and stylesheets. These are added as resources and dependencies on packaging. It also detects locallinks in the template and adds the linked Node as a dependency.
-
-### UltimatePicker
-* **Full name:** `Umbraco.Courier.DataResolvers.UltimatePicker`
-* **Triggers on:**  Propertydata, which have a UltimatePicker type as data type
-* Selected node IDs are converted to GUIDs and the linked nodes are added as dependencies
-
-
-### Upload
-* **Full name:** `Umbraco.Courier.DataResolvers.Upload`
-* **Triggers on:**  Propertydata, which have a Upload type as data type
-* If upload field contains a file, the file is added as a resource on the document.
-
-##### Configuration
-
-	<files>
-	    <!-- add new datatype elements for data types that stores files as a path ex: /meda/223/file.png -->
-	    <add key="Upload">5032a6e6-69e3-491d-bb28-cd31cd11086c</add>
-	    <add key="Ucomponents-Filepicker">318a9c2e-3966-4979-8c1d-575c5d5f669b</add>
-	</files>
-
-### UsercontrolWrapper
-* **Full name:** `Umbraco.Courier.DataResolvers.UsercontrolWrapper`
-* **Triggers on:**  The UsercontrolWrapper Data types
-*If the data type has a .ascx file selected as render, the file is added as a resource to ensure it is transferred with the data type.
+
```

Comment:
Seems you have simply removed the headlines and explanation here, not sure why? 🙂